src/main/java/frc/robot/subsystems/scoring/AimerIOTalon.java

```diff
@@ -126,12 +129,81 @@ public void setFF(double kS, double kV, double kA, double kG) {
         aimerRight.getConfigurator().apply(slot0);
     }
 
+    private boolean checkForAimMotorFailure() {
+
+        // LEFT MOTOR
+        if (motorFailureCheckPair(aimerLeft, aimerRight)) {
+            motorLeftFailure = true;
+        }
+
+        // RIGHT MOTOR
+        if (motorFailureCheckPair(aimerRight, aimerLeft)) {
+            motorRightFailure = true;
+        }
+
+        if (!motorCheckOverriden) {
+            shutOffFaultyAimMotors();
+        }
+
+        return motorLeftFailure || motorRightFailure;
+    }
+
+    private boolean motorFailureCheckPair(TalonFX check, TalonFX compare) {
+        if (!check.isAlive()) {
+            // motor no longer communicating with robot
+            return true;
+        }
+
+        if (Math.abs(compare.getStatorCurrent().getValueAsDouble())
+                                - Math.abs(check.getStatorCurrent().getValueAsDouble())
+                        > 1
+                && Math.abs(check.getMotorVoltage().getValueAsDouble()) > 0.5) {
+            // motor voltage is really small when it shouldn't be
+            return true;
+        }
+
+        if (Math.abs(check.getStatorCurrent().getValueAsDouble())
+                                - Math.abs(compare.getStatorCurrent().getValueAsDouble())
+                        > 1
+                && Math.abs(check.getMotorVoltage().getValueAsDouble()) < 0.5) {
```

Comment:
Some points of feedback here:
1. These thresholds should be set as constants in Constants.java
2. These are really tight thresholds. 1 A or 0.5 V is common to see as a difference between two motors. Would recommend a much bigger value for amps, but this should be pulled from match/robot data where the aimer motor was not connected most likely.
3. We should add a time aspect to this. "If the motors disagree by X current for 1 cycle" will yield many false positives. Let's add that they must disagree for T time
4. Some of this logic looks duplicated?